js/start.js

```diff
@@ -351,33 +360,84 @@ function start() {
       fetchStartupData().done(() => {
         app.pageNav.navigable = true;
         app.pageNav.setAppProfile();
+        getChatContainer().removeClass('hide');
```

Comment:
For the reason I mentioned in stand-up, I'm very leery of giving any modal a z-index. It opens up a can of worms that I think is much better avoided.

I don't think it's a problem to hide / show chat because it's done in only one circumstance. If there's no connection, it's hidden, otherwise it's visible and never obscured. I believe the UX req is that the chat should never be hidden, unless there's no connection in which case it's not usable.